10.2/root/etc/my.cnf.d/utf8.cnf

```diff
@@ -0,0 +1,4 @@
+[mysqld]
```

Comment:
That's a deep question there so I'll ramble through some of my thinking.

There are benefits to calling it out explicitly with some commentary about why it's there from an educational perspective. The cost is that the config has to be reviewed to ensure defaults haven't evolved to better values periodically. It's a minor cost though and practically it gets incurred very rarely and usually just as part of troubleshooting something else.

We don't have a declared approach to "we make all the config values explicit even when using defaults" versus "we only change things where the defaults aren't good" though I think we tend to primarily tend towards the latter.

I'd vote for removing it in 10.2 but it's like a 51/49 decision so if your weighting swings it the other way (even if for consistency with the config in other versions) I can live with it.